source/Coop/Coop.csproj

```
@@ -282,38 +212,35 @@
       <HintPath>..\..\mb2\bin\Win64_Shipping_Client\TaleWorlds.TwoDimension.Standalone.dll</HintPath>
     </Reference>
   </ItemGroup>
-  <ItemGroup>
-    <Compile Include="AssemblyHellscape.cs" />
-    <Compile Include="Lib\NoHarmony\NoHarmonyLoader.cs" />
-    <Compile Include="CoopMod.cs" />
-    <Compile Include="Properties\AssemblyInfo.cs" />
-    <Compile Include="UI\CharacterCreationUI\CharacterCreationUI.cs" />
-    <Compile Include="UI\CoopConnectionUI.cs" />
-    <Compile Include="UI\CoopConnectMenuVM.cs" />
-    <Compile Include="UI\LoadGameUI\CoopLoadUI.cs" />
-  </ItemGroup>
-  <ItemGroup>
-    <None Include="App.config" />
-    <None Include="packages.config" />
-  </ItemGroup>
   <ItemGroup>
     <ProjectReference Include="..\Common\Common.csproj">
-      <Project>{9A045658-9A0C-47E5-A1D9-E1B45E85FD0D}</Project>
+      <Project>{e474a5b5-3f73-46cb-b68c-e8fa5199950b}</Project>
       <Name>Common</Name>
     </ProjectReference>
     <ProjectReference Include="..\Coop.Core\Coop.Core.csproj">
-      <Project>{442b73f2-b103-4ef6-8b4b-a2f36f169515}</Project>
+      <Project>{442B73F2-B103-4EF6-8B4B-A2F36F169515}</Project>
       <Name>Coop.Core</Name>
     </ProjectReference>
     <ProjectReference Include="..\GameInterface\GameInterface.csproj">
       <Project>{D22CB8FA-40CC-4F47-986F-1C5A4415980B}</Project>
       <Name>GameInterface</Name>
     </ProjectReference>
   </ItemGroup>
-  <Import Project="$(MSBuildToolsPath)\Microsoft.CSharp.targets" />
-  <PropertyGroup>
-    <PostBuildEvent>powershell -ExecutionPolicy Unrestricted ^
-            -File "$(SolutionDir)..\deploy.ps1" -SolutionDir "$(SolutionDir)\" ^
-            -TargetDir "$(SolutionDir)Coop\bin\$(Configuration)\\"</PostBuildEvent>
-  </PropertyGroup>
+  <ItemGroup>
+    <PackageReference Include="DistributedLock.FileSystem" Version="1.0.0" />
+    <PackageReference Include="Microsoft.Bcl.AsyncInterfaces" Version="5.0.0" />
+    <PackageReference Include="MonoMod.RuntimeDetour" Version="22.7.31.1" />
+    <PackageReference Include="MonoMod.Utils" Version="22.7.31.1" />
+    <PackageReference Include="Serilog.Enrichers.Process" Version="2.0.2" />
+    <PackageReference Include="Serilog.Sinks.Debug" Version="2.0.0" />
+    <PackageReference Include="Serilog.Sinks.File" Version="5.0.0" />
+    <PackageReference Include="System.Threading.Tasks.Extensions" Version="4.5.4" />
+  </ItemGroup>
+
+  <Target Name="RunDeploy" AfterTargets="ReBuild">
+    <Exec Condition=" '$(OS)' == 'Windows_NT' " 
+          Command='powershell -ExecutionPolicy Unrestricted ^
+          -File "$(SolutionDir)..\deploy.ps1" -SolutionDir "$(SolutionDir)" ^
+          -TargetDir "$(SolutionDir)Coop\bin\$(Configuration)\netstandard2.0\"'/>
+  </Target>
```

This currently doesn't work on my windows machine, update to the following
```suggestion
  <Target Name="PostBuild" AfterTargets="PostBuildEvent">
    <Exec Condition=" '$(OS)' == 'Windows_NT' " Command="powershell -ExecutionPolicy Unrestricted -File &quot;$(SolutionDir)\..\deploy.ps1&quot; -SolutionDir &quot;$(SolutionDir)\&quot; -TargetDir &quot;$(SolutionDir)\Coop\bin\$(Configuration)\netstandard2.0\&quot;" />
  </Target>
```

---

interesting, works on mine. fixed